Makefile

```diff
@@ -3,12 +3,12 @@
 init_broker:	
 	@echo command | date
 	@echo Initializing Kafka broker
-	docker-compose -f docker-compose.yml up -d
+	docker compose -f docker-compose.yml up -d
```

Comment:
Fixed the makefile, since it was failing to build due to using the wrong command under latest Ubuntu